src/test/java/com/softserve/academy/spaced/repetition/security/JwtTokenUtilTest.java

```diff
@@ -90,6 +91,7 @@ public void generateToken() throws Exception {
         PowerMockito.doCallRealMethod().when(jwtTokenUtil, "generateToken", anyObject());
         PowerMockito.doCallRealMethod().when(jwtTokenUtil, "generateExpirationDate");
         Whitebox.setInternalState(jwtTokenUtil, "secret", "mySecret");
+        Whitebox.setInternalState(jwtTokenUtil, "expiration", 1000L);
```

Comment:
Why 1000L?

Reply:
:)

Reply:
The expiration field is of type Long.

1000L is to indicate that it is a value of type long, not int.

Autoboxing will take care of converting 1000L of type long to Long.

Reply:
Yes, I know that, thanks, but I don't understand, why exactly 1000?

Reply:
It is just test data. Any value to avoid NPE inside com.softserve.academy.spaced.repetition.security.JwtTokenUtil#generateExpirationDate

Reply:
Okay, thank you! :)